src/Packages/Microsoft.CodeDom.Providers.DotNetCompilerPlatform/Microsoft.CodeDom.Providers.DotNetCompilerPlatform.nuproj

```diff
@@ -29,9 +29,24 @@
         <NuGetContent Include="build\*">
             <Destination>build\net45</Destination>
         </NuGetContent>
+        <NuGetContent Include="build\*">
+            <Destination>build\net46</Destination>
+        </NuGetContent>
+        <NuGetContent Include="build\net45\*">
+            <Destination>build\net45</Destination>
+        </NuGetContent>
+        <NuGetContent Include="build\net46\*">
+            <Destination>build\net46</Destination>
```

Comment:
I am assuming this the latest you are referring later?  So this should cover anything after 4.6, correct?  If so, do you want to use latest here as well?

Reply:
net46 folder name has special meaning in Nupkg. Check [this](https://docs.microsoft.com/en-us/nuget/schema/target-frameworks).